### PR TITLE
THRIFT-4694: Upgrade Java to Java 1.8

### PR DESCRIPTION
**Pull request by BELUGABEHR**
Some places reference JDK 6, others JDK 7.  Upgrade all to JDK 8

### CODE REVIEW

File: doc/install/README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ These are only required if you choose to build the libraries for the given langu
     * libevent (optional, to build the nonblocking server)
     * zlib (optional)
 * Java
-    * Java 1.7
+    * Java 1.8
     * Apache Ant
```
Comment (jeking3) [inline doc/install/README.md]: Since the build changed to gradle I'm not certain we need ant any more.

**Comment by BELUGABEHR** [comment]
Hello @jeking3  Thanks for the look over!

So, interestingly, the LANGUAGES.md file already states that Thrift supports "1.8.0_151".

I reviewed the Docker build files and they do reflect the update to Java 1.8.

In regards to Ant, I would like to request that we table that for a different JIRA in order to more thoroughly examine if it is still a dependent.  I see it's still being installed as a dependency in the Docker container, so it may still be needed.  Either way, this should be done in a new JIRA.

**Comment by BELUGABEHR** [comment]
https://issues.apache.org/jira/browse/THRIFT-4701

**Comment by jeking3** [comment]
There is a conflict, a file was removed - please rebase on master.

**Comment by jeking3** [comment]
When you rebase you want to do this:

* git checkout master
* git pull
* git checkout <your branch>
* git rebase master
* git push -u <your personal fork name> <your branch> --force

**Comment by BELUGABEHR** [comment]
Thanks @jeking3 ! Ya, I'm still getting my head around git, GitHub is helpful to verify.  I knew it was wrong. :)

**Pull request merged**